### PR TITLE
osdocs-399 updating RHCOS locations for GA

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://jira.coreos.com/browse/OSDOCS-399

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@vikram-redhat, does this need QE?

**Comment by vikram-redhat** [comment]
Yes this needs QE approval.

@xltian we need an exception for this to be included for GA. Could you please assign someone to QE this?

File: modules/installation-vsphere-machines.adoc [hunk 1]
```diff
@@ -71,11 +64,14 @@ $ base64 -w0 <installation_directory>/worker.ign > <installation_directory>/work
 $ base64 -w0 <installation_directory>/append-bootstrap.ign > <installation_directory>/append-bootstrap.64
 ----
 
-. Obtain the {op-system} OVA image:
+. Obtain the {op-system} OVA image from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal or the
+link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.1/[{op-system} image mirror]
```
Comment (jianlinliu) [inline modules/installation-vsphere-machines.adoc]: I saw there is nothing for GA version yet, is there is some available preview and downloadable rhcos image?

**Comment by jianlinliu** [comment]
@morenod Could you help check BM part?

**Comment by morenod** [comment]
@vikram-redhat @kalexand-rh reviewed BM part, looks good to me

File: modules/installation-vsphere-machines.adoc (hunk 1 above)
Comment (jianlinliu) [reply modules/installation-vsphere-machines.adoc]: If the link only point user to rhcos parent folder, but not some specific version of folder, customer need navigate to proper folder manually, I am OKAY to merge it.

**Pull request merged**